functions/feynman-mvp-firebase-adminsdk-d10vx-70c5b69675.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This compromises security and so needs to be wiped and sent to the gulag archipelago